### ISSUE TITLE
[Request] Cycle triggers via drop-down for winlator

### ISSUE THREAD

**Issue by binhex** [langerhans/OdinTools#38]
Hi, so I have been using winlator a fair bit recently and one of the things I need to do for certain games (farcry and fallout/4) is to set the triggers to digital, failure to do this will need to issues using the triggers where for instance, the gun will not fire or will fire intermittently, setting to digital solves this problem.

This request is to add in a icon to the android drop-down panel to cycle the trigger type digital/analogue/both.

**Comment by miguelguzmanr** [comment]
Hello! Thank you for opening an issue.
Perhaps you're looking for `L2/R2 mode` quick settings tile.
It allows the user to switch the triggers' behavior between digital, analog or "both", and sounds fairly similar to what you're looking for.
Please let me know if that solves your issue!

![image](https://github.com/langerhans/OdinTools/assets/40463442/42015e6e-c42f-4ce4-8236-4d64d2c5f501)

**Comment by binhex** [comment]
Sorry for the delay in getting back to you, and yes the quick settings tile looks t suffice, and with a bit more digging i realised that you can also set the trigger from digital/analogue/both via the odin swipe menu on the right, this has the added benefit that it then changes the trigger type for the active application, which in my case is winlator which is excellent, as it then means i dont need to switch it back to 'both' when firing up another more classic emulator such as retroarch.

So in short, yes im going to close this off and i aplogise for raising a non-issue, all good nothing to see here :-)
